### PR TITLE
[FIX] Clarify Upper-casing of Channels.tsv Channel Type

### PR DESCRIPTION
**Pull request by adam2392**
Closes: #534 

related validator PR: https://github.com/bids-standard/bids-validator/pull/1022

### CODE REVIEW

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ and EEG modality; however, only types that are common in iEEG data are listed he
 | REF      | Reference channel                                                      |
 | OTHER    | Any other type of channel                                              |
 
-The free text field for the channel description can for example be specified as
+The free text field (i.e. `OTHER` channel type) for the channel description can for example be specified as
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:296]:
```suggestion
The free text field (i.e., `OTHER` channel type) for the channel description can for example be specified as
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 2]
```diff
@@ -268,7 +268,7 @@ ECG1  ECG   uV    n/a         0.11        good    n/a
 TR1   TRIG  n/a   n/a         n/a         good    n/a
 ```
 Restricted keyword list for field type in alphabetic order (shared with the MEG
-and EEG modality; however, only types that are common in iEEG data are listed here):
+and EEG modality; however, only types that are common in iEEG data are listed here). Note that upper-casing is REQUIRED:
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:271]:
```suggestion
and EEG modality; however, only types that are common in iEEG data are listed here). Note that upper-case is REQUIRED:
```

File: src/04-modality-specific-files/03-electroencephalography.md [hunk 3]
```diff
@@ -236,7 +236,7 @@ SHOULD be present:
 | status_description | OPTIONAL. Free-form text description of noise or artifact affecting data quality on the channel. It is meant to explain why the channel was declared bad in `[status]`.                                                                                                       |
 
 Restricted keyword list for field `type` in alphabetic order (shared with the
-MEG and iEEG modality; however, only the types that are common in EEG data are listed here):
+MEG and iEEG modality; however, only the types that are common in EEG data are listed here). Note that upper-casing is REQUIRED:
```
Comment (sappelhoff) [inline src/04-modality-specific-files/03-electroencephalography.md:239]:
```suggestion
MEG and iEEG modality; however, only the types that are common in EEG data are listed here). Note that upper-case is REQUIRED:
```

File: src/04-modality-specific-files/02-magnetoencephalography.md [hunk 4]
```diff
@@ -226,7 +226,7 @@ UDIO001 TRIG V analogue trigger 1200  0.1 300 0 n/a good
 MLC11 MEGGRADAXIAL T sensor 1st-order grad 1200 0 n/a 50 SSS bad
 ```
 
-Restricted keyword list for field `type`
+Restricted keyword list for field `type`. Note that upper-casing is REQUIRED:
```
Comment (sappelhoff) [inline src/04-modality-specific-files/02-magnetoencephalography.md:229]:
```suggestion
Restricted keyword list for field `type`. Note that upper-case is REQUIRED:
```

File: src/04-modality-specific-files/02-magnetoencephalography.md [hunk 5]
```diff
@@ -226,7 +226,7 @@ UDIO001 TRIG V analogue trigger 1200  0.1 300 0 n/a good
 MLC11 MEGGRADAXIAL T sensor 1st-order grad 1200 0 n/a 50 SSS bad
 ```
 
-Restricted keyword list for field `type`
+Restricted keyword list for field `type`. Note that upper-case is REQUIRED:
```
Comment (sappelhoff) [inline src/04-modality-specific-files/02-magnetoencephalography.md:229]:
```suggestion
Restricted keyword list for field `type`.
Note that upper-case is REQUIRED:
```

File: src/04-modality-specific-files/03-electroencephalography.md [hunk 6]
```diff
@@ -236,7 +236,7 @@ SHOULD be present:
 | status_description | OPTIONAL. Free-form text description of noise or artifact affecting data quality on the channel. It is meant to explain why the channel was declared bad in `[status]`.                                                                                                       |
 
 Restricted keyword list for field `type` in alphabetic order (shared with the
-MEG and iEEG modality; however, only the types that are common in EEG data are listed here):
+MEG and iEEG modality; however, only the types that are common in EEG data are listed here). Note that upper-case is REQUIRED:
```
Comment (sappelhoff) [inline src/04-modality-specific-files/03-electroencephalography.md:239]:
```suggestion
MEG and iEEG modality; however, only the types that are common in EEG data are listed here).
Note that upper-case is REQUIRED:
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 7]
```diff
@@ -293,7 +293,7 @@ and EEG modality; however, only types that are common in iEEG data are listed he
 | REF      | Reference channel                                                      |
 | OTHER    | Any other type of channel                                              |
 
-The free text field for the channel description can for example be specified as
+The free text field (i.e., `OTHER` channel type) for the channel description can for example be specified as
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:296]:
```suggestion
The free text field `description` in the `*_channels.tsv` file can for example be specified as
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 8]
```diff
@@ -293,7 +293,7 @@ and EEG modality; however, only types that are common in iEEG data are listed he
 | REF      | Reference channel                                                      |
 | OTHER    | Any other type of channel                                              |
 
-The free text field for the channel description can for example be specified as
+The free text field `description` in the `*_channels.tsv` file can for example be specified as
 intracranial, stimulus, response, vertical EOG, horizontal EOG, skin
 conductance, eyetracker, etc.
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:298]:
```suggestion
Example of free-form text for field `description`

-   intracranial, stimulus, response, vertical EOG,  skin conductance
```

File: src/04-modality-specific-files/02-magnetoencephalography.md [hunk 9]
```diff
@@ -259,7 +260,7 @@ Restricted keyword list for field `type`
 | FITERR           | Fit error signal from each head localization coil    |
 | OTHER            | Any other type of channel                            |
 
-Example of free text for field `description`
+Example of free text for field `description`.
```
Comment (sappelhoff) [inline src/04-modality-specific-files/02-magnetoencephalography.md:263]:
```suggestion
Example of free text for field `description`:
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 10]
```diff
@@ -268,7 +268,7 @@ ECG1  ECG   uV    n/a         0.11        good    n/a
 TR1   TRIG  n/a   n/a         n/a         good    n/a
 ```
 Restricted keyword list for field type in alphabetic order (shared with the MEG
-and EEG modality; however, only types that are common in iEEG data are listed here):
+and EEG modality; however, only types that are common in iEEG data are listed here). Note that upper-case is REQUIRED:
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:271]:
```suggestion
and EEG modality; however, only types that are common in iEEG data are listed here).
Note that upper-case is REQUIRED:
```

File: src/04-modality-specific-files/04-intracranial-electroencephalography.md [hunk 11]
```diff
@@ -293,9 +293,9 @@ and EEG modality; however, only types that are common in iEEG data are listed he
 | REF      | Reference channel                                                      |
 | OTHER    | Any other type of channel                                              |
 
-The free text field for the channel description can for example be specified as
-intracranial, stimulus, response, vertical EOG, horizontal EOG, skin
-conductance, eyetracker, etc.
+Example of free-form text for field `description`
```
Comment (sappelhoff) [inline src/04-modality-specific-files/04-intracranial-electroencephalography.md:296]:
```suggestion
Example of free-form text for field `description`:
```

**Comment by hoechenberger** [comment]
> . I should have just checked out the PR and done the changes locally but was too lazy 👼

I feel you 😅☺️

**Pull request merged**